src/app/app.module.ts.orig

```diff
@@ -15,23 +35,56 @@ import {AngularFireModule} from '@angular/fire/compat';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 //Coponents for SignupComponent
+
 import { MatToolbarModule } from '@angular/material/toolbar';
 import { MatIconModule } from '@angular/material/icon';
 import { MatButtonModule } from '@angular/material/button';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatInputModule } from '@angular/material/input';
 import { ReactiveFormsModule } from '@angular/forms';
 import { MatMenuModule} from '@angular/material/menu';
+
+import {CoreModule} from "./core/core.module";
+import {SharedModule} from "./shared/shared.module";
+
+import { NavbarComponent } from './core/navbar/navbar.component';
+import { MyInboxComponent } from './private/pages/myInbox/my-inbox.component';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
```

Comment:
For code organization, code is suggested to be moved to where all the Mat imports are as well.